packages/angular/.npmignore

```diff
@@ -1,4 +1,5 @@
 *
 !/dist/**/*
 !/esm/**/*
+!/build/types/**/*
 *.tsbuildinfo
```

Comment:
As part of your "make the .`npmignore`s match" work here, I think we should get rid of these `*.tsbuildinfo` entries, which show up in some but not all `.npmignore` files, since they're redundant given the `*`.

Reply:
Good point. I was wondering if there was some reason why this file was explicitly ignored but it's probably just a leftover from previous configs... I removed it in all `.npmignore`s and checked if it made any difference. Couldn't find any.